docs/package.json

```diff
@@ -5,7 +5,7 @@
   "author": "MUI Team",
   "license": "MIT",
   "scripts": {
-    "build": "rimraf ./export && cross-env NODE_ENV=production next build --profile && pnpm build-sw",
+    "build": "rimraf ./export && cross-env NODE_ENV=production next build && pnpm build-sw",
```

Comment:
@eps1lon initially added this flag in https://github.com/mui/material-ui/pull/21975. Making the connection as it seems that this started with https://github.com/mui/mui-x/pull/15144.